unittests/DxilHash/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(LLVM_LINK_COMPONENTS
+  Support
+  DxilHash
+  )
+
+add_clang_unittest(DxilHashTests
```

Comment:
Do these tests get automatically picked up by our test runners?

Reply:
Yes.
It will be picked up as part of LLVM-Unit.